t/plugin/http-logger.t

```diff
@@ -108,7 +108,43 @@ done
 
 
 
-=== TEST 4: add plugin
+=== TEST 4: check log schema(include_resp_body_expr)
```

Comment:
Could we add the test at the end, so the other test numbers won't be changed? We can use `reindex` to fix the number.

Reply:
updated